generator/src/main/java/com/scottlogic/deg/generator/CommandLine/ValidationResult.java

```diff
@@ -0,0 +1,15 @@
+package com.scottlogic.deg.generator.CommandLine;
```

Comment:
This looks like it could have bigger scope that just the command line - suggest creating a "validators" folder?